content/dependencies.md

```diff
@@ -1,12 +1,30 @@
 ## II. Dependencies
 ### Explicitly declare and isolate dependencies
 
-Most programming languages offer a packaging system for distributing support libraries, such as [CPAN](http://www.cpan.org/) for Perl or [Rubygems](http://rubygems.org/) for Ruby.  Libraries installed through a packaging system can be installed system-wide (known as "site packages") or scoped into the directory containing the app (known as "vendoring" or "bundling").
+#### 1\. A twelve-factor app is built and run in deterministic environments.
 
-**A twelve-factor app never relies on implicit existence of system-wide packages.**  It declares all dependencies, completely and exactly, via a *dependency declaration* manifest.  Furthermore, it uses a *dependency isolation* tool during execution to ensure that no implicit dependencies "leak in" from the surrounding system.  The full and explicit dependency specification is applied uniformly to both production and development.
```

Comment:
I note that this one specifically called out a *dependency declaration* manifest. Do we want to call out the manifest in the new version?